crema/choices.py

```diff
@@ -117,6 +119,8 @@ class EventPartition(Enum):
 
     # AMS
     ASSESSMENT = 16
+    # topic for master user assessment data
+    USER_ASSESSMENT = 16
```

Comment:
how many events are we expecting here?

Reply:
probably same or a little more as the number of new users...